src/script/cpp_api/s_player.cpp

```diff
@@ -235,6 +235,30 @@ void ScriptApiPlayer::on_authplayer(const std::string &name, const std::string &
 	runCallbacks(3, RUN_CALLBACKS_MODE_FIRST);
 }
 
+void ScriptApiPlayer::on_wielditem_change(ServerActiveObject *player,
+	const ItemStack &old_item, u16 old_wield_idx,
+	const ItemStack &new_item, u16 new_wield_idx)
+{
+	SCRIPTAPI_PRECHECKHEADER
+
+	// Get core.registered_on_wielditem_change callbacks
+	lua_getglobal(L, "core");
+	lua_getfield(L, -1, "registered_on_wielditem_change");
+
+	// First param: player object
+	objectrefGetOrCreate(L, player);
+
+	// Params 2 & 3: Old wielded item and its wield index
```

Comment:
This doesn't match the documentation, did you forget to update it?

Reply:
Indeed. Updating docs...

Reply:
Done

Reply:
Still missing...

Reply:
I'm absolutely sure I updated this... o_O

Reply:
Ah, I didn't stage the change before committing, and switched to another branch... :facepalm: 

Updated docs, sorry. :)